compiler/AST/primitive.cpp

```diff
@@ -45,13 +45,13 @@ returnInfoBool(CallExpr* call) {
 }
 
 static Type*
-returnInfoString(CallExpr* call) {
-  return dtString;
+returnInfoStringC(CallExpr* call) {
+  return dtStringC;
 }
 
 static Type*
-returnInfoStringC(CallExpr* call) {
-  return dtStringC;
+returnInfoString(CallExpr* call) {
+  return dtString;
```

Comment:
Are my eyes deceiving me, or is this diff just the result of two routines being swapped in order?  Could we swap back to minimize diffs?